core/app/models/concerns/spree/product_scopes.rb

```diff
@@ -305,6 +305,8 @@ def self.for_user(user = nil)
         def self.search_by_name(query)
           if defined?(SpreeGlobalize)
             joins(:translations).order(:name).where("LOWER(#{Product::Translation.table_name}.name) LIKE LOWER(:query)", query: "%#{query}%").distinct
+          elsif defined?(Mobility)
```

Comment:
Is there a case where Mobility is not defined? Isn't it bundled by default?

Reply:
Theoretically, someone may want to disable the mobility gem.  One `if` statement is not a big deal.